clients/drcachesim/tools/filter/record_filter.cpp

```diff
@@ -97,11 +96,7 @@ parse_string(const std::string &s, char sep = ',')
     do {
         pos = s.find(sep, at);
         unsigned long long parsed_number = std::stoull(s.substr(at, pos));
-        // Check that T can hold the parsed value. Otherwise, skip it.
-        if (parsed_number >= std::numeric_limits<T>::min() &&
-            parsed_number <= std::numeric_limits<T>::max()) {
-            vec.push_back(static_cast<T>(parsed_number));
-        }
+        vec.push_back(static_cast<T>(parsed_number));
```

Comment:
Add a comment: `XXX: the value may be truncated if T is not large enough. Maybe throw a warning? But how do we detect limits for enums since std::numeric_limits returns zero.`.